substrate/cmd/delete.go

```diff
@@ -24,18 +24,16 @@ import (
 	"knative.dev/pkg/logging"
 )
 
-func init() {
-	rootCmd.LocalFlags().StringVar(&options.File, "name", "", "Name for the environment")
-	rootCmd.LocalFlags().StringVarP(&options.File, "file", "f", "", "Configuration file for the environment")
-	rootCmd.AddCommand(&cobra.Command{
+func deleteCommand() *cobra.Command {
```

Comment:
I like calling this delete. Makes me think the other command should be "create" instead of "bootstrap"

Reply:
`delete` conflicts with the delete function call made as part of this command

Reply:
Oh I'm not suggesting calling it delete(). I'm just saying that I like your naming of `kitctl delete`